Services/Administration/OJS.Services.Administration.Business/Extensions/AdminActionContextExtensions.cs

```diff
@@ -12,8 +12,17 @@ namespace OJS.Services.Administration.Business.Extensions;
 
 public static class AdminActionContextExtensions
 {
-    public static string GetFormValue(this AdminActionContext actionContext, AdditionalFormFields field)
-        => actionContext.EntityDict[field.ToString()];
+    public static string? GetFormValue(this AdminActionContext actionContext, AdditionalFormFields field)
+    {
+        try
+        {
+            return actionContext.EntityDict[field.ToString()];
+        }
+        catch (KeyNotFoundException)
+        {
+            return null;
+        }
```

Comment:
Can you use `GetValueOrDefault` extension method on the Dictionary, or use `HasKey`, instead of catching exception?